src/lib/tvgCanvasImpl.h

```diff
@@ -64,13 +64,14 @@ struct Canvas::Impl
         if (!renderer || !renderer->clear()) return Result::InsufficientCondition;
 
         //Free paints
-        for (auto paint : paints) {
-            paint->pImpl->dispose(*renderer);
-            if (free) delete(paint);
+        if (free) {
+            for (auto paint : paints) {
+                paint->pImpl->dispose(*renderer);
```

Comment:
We could use `clear(false)` if the paint object was managed externally.
Um.. maybe, paint's dispose is not called separately in paint's destructor, so I concerned about memory leaks in this part. Are there other alternatives without breaking backwards compatibility?

Reply:
dispose() is changed again.

Upgrading to a major version provides an good opportunity to eliminate poor APIs or usages. 
The clear(false) method has demonstrated this need, as indicated by a @warning.
When we have valid reasons, we make changes. This decision is consistent with our policy.

Let's see.